uiengine/src/main/java/com/axway/ats/uiengine/SwingDriver.java

```diff
@@ -100,6 +100,20 @@ public SwingDriver( String jnlpLocation, boolean cacheEnabled ) {
         this.isJnlpCached = cacheEnabled;
     }
 
+    /**
+     *
+     * @param jnlpLocation JNLP location. It can be local JNLP file path or a remote url
+     * @param windowTitle the window title
+     * @param cacheEnabled whether the JNLP is cached or not
+     */
+    public SwingDriver( String jnlpLocation, String windowTitle, boolean cacheEnabled ) {
+
+        this.jnlpLocation = jnlpLocation;
+        this.windowTitle = windowTitle;
```

Comment:
Won't this be replaced with the value obtained by invoking the method jnlpAppInstance.getTitle();

in the start method of this class?